distributor/distributor.go

```diff
@@ -441,61 +434,84 @@ func (d *Distributor) Query(ctx context.Context, from, to model.Time, matchers .
 			return err
 		}
 
-		if len(ingesters) < d.cfg.MinReadSuccesses {
-			return fmt.Errorf("could only find %d ingesters for query. Need at least %d", len(ingesters), d.cfg.MinReadSuccesses)
+		// We need a response from a quorum of ingesters, which is n/2 + 1.
+		minSuccess := (len(ingesters) / 2) + 1
```

Comment:
s/DynamoDB/Dynamo/

But yes, that is what the paper says.  In our case, we set r = w = n / 2 + 1 (although the +1 is due to integer arithmetic).  So `r+w > n` -> `2r > n` -> `2n/2 + 2 > n`  -> `n + 2 > 2`